SWLOR.Game.Server/Scripts/Delayed/CraftCreateItem.cs

```diff
@@ -272,9 +278,10 @@ private float CalculateEquipmentBonus(NWPlayer player, SkillType skillType)
                 case SkillType.Engineering: equipmentBonus = effectiveStats.Engineering; break;
                 case SkillType.Fabrication: equipmentBonus = effectiveStats.Fabrication; break;
                 case SkillType.Medicine: equipmentBonus = effectiveStats.Medicine; break;
+                case SkillType.Lightsaber: equipmentBonus = ((sabercraft / 10) + force); break;
```

Comment:
I'm fine with the atmosphere thing, gives a good bonus. I'm less okay with an equipment bonus being affected by skill. Let's remove this line for now since you're going to be making changes to lightsaber crafting anyway.